astra-sim/workload/Workload.cc

```diff
@@ -151,127 +158,146 @@ void Workload::issue_remote_mem(shared_ptr<Chakra::ETFeederNode> node) {
   WorkloadLayerHandlerData* wlhd = new WorkloadLayerHandlerData;
   wlhd->sys_id = sys->id;
   wlhd->workload = this;
-  wlhd->node_id = node->getChakraNode()->id();
-  sys->remote_mem->issue(node->getChakraNode()->tensor_size(), wlhd);
+  wlhd->node_id = node->id();
+  sys->remote_mem->issue(node->tensor_size(), wlhd);
 }
 
 void Workload::issue_comp(shared_ptr<Chakra::ETFeederNode> node) {
```

Comment:
Note, I think we should rename this function to something more meaningful if I understand these changes correctly.  This function is not just for computation layers.

Reply:
This function is only for compute operators.